README.md

```diff
@@ -1,31 +1,113 @@
+</div>
+<br>
 
+| Information | Links |
+| :---------- | :-----|
+|   Project   | [![License](https://img.shields.io/badge/License-BSD%203--Clause-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://opensource.org/licenses/BSD-3-Clause) [![Nebari documentation](https://img.shields.io/badge/%F0%9F%93%96%20Read-the%20docs-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://nebari-docs.netlify.app/) [![PyPI version](https://badge.fury.io/py/qhub.svg)](https://badge.fury.io/py/qhub) |
+|  Community  | [![GH discussions](https://img.shields.io/badge/%F0%9F%92%AC%20-Participate%20in%20discussions-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://github.com/nebari-dev/nebari/discussions) [![Open an issue](https://img.shields.io/badge/%F0%9F%93%9D%20Open-an%20issue-gray.svg?colorA=2D2A56&colorB=5936D9&style=flat.svg)](https://github.com/nebari-dev/nebari/issues/new/choose) |
+|     CI      | [![Kubernetes Tests](https://github.com/Quansight/qhub/actions/workflows/kubernetes_test.yaml/badge.svg)](https://github.com/Quansight/qhub/actions/workflows/kubernetes_test.yaml) [![Tests](https://github.com/Quansight/qhub/actions/workflows/test.yaml/badge.svg)](https://github.com/Quansight/qhub/actions/workflows/test.yaml) |
+| Website | [![Website Deploy](https://github.com/nebari-dev/nebari/actions/workflows/deploy.yml/badge.svg)](https://github.com/nebari-dev/nebari/actions/workflows/deploy.yml) 
 
-🏗 An opinionated open source JupyterHub distribution
+---
 
-## Documentation
+Nebari is an customizable Data Science and MLOPs Platform designed as a managed integration of open source technologies. 
+It uses an infrastructure-as-code approach to quickly and easily deploy a shared data science environment, on premises 
+or in the cloud of choice. 
 
-### Installing  `node`
-- Checking if node is already present in the system (should print the version number)
-    ```bash
-     node -v
-    ```
-- To install node visit [nodejs.org](https://nodejs.org/en/download/)
+Nebari is an opinionated open source JupyterHub distribution with a variety of common integrations to help you and your 
+team get set up with all your favorite tools ...  and a few you didn't know you needed! 
 
-### Building the site
+Check out our full [documentation](https://nebari-docs.netlify.app/)!
 
-Install the neccesary packages
-```bash
-cd docs
-npm install
-```
+---
+
+## Table of contents
+
+- [Nebari 101](#nebari-101)
+- [Nebari on-prem](#nebari-on-prem)
+  <!-- - [:cloud: Cloud Providers](#cloud-cloud-providers) -->
+- [:computer: Installation](#computer-installation)
+- [:question: Questions?](#question-questions)
+- [:book: Code of Conduct](#book-code-of-conduct)
+- [:gear: Installing the Development version of Nebari](#gear-installing-the-development-version-of-nebari)
+- [:raised_hands: Contributions](#raised_hands-contributions)
+- [License](#license)
+
+## Nebari 101
+
+Nebari Cloud deployments use Kubernetes and are is built using 
```

Comment:
```suggestion
Nebari Cloud can be deployed on any Kubernetes cluster by leveraging these tools:
```
Do we refer to Nebari as `Nebari Cloud` elsewhere?

Reply:
I don't think we refer to `Nebari Cloud` in these docs. For a while we had `qhub cloud` and `qhub on-prem`. I'll just remove the term `cloud` here since we mention the on-prem option as a separate repo